app/services/word-service.js

```diff
@@ -0,0 +1,27 @@
+import Ember from 'ember';
+import PastWord from 'lyricsss/models/past-word';
+
+export default Ember.Service.extend({
+  add(word, team, answerCorrect, time) {
```

Comment:
we could change this function's inputs to be a single hash with these four variables named?

`add({word, team, answerCorrect, time}}` will automatically "destructure" it into variables with those names too.